src/main/java/org/isf/menu/gui/MainMenu.java

```diff
@@ -67,23 +64,20 @@
 public class MainMenu extends JFrame implements ActionListener, Login.LoginListener, SubMenu.CommandListener {
 
 	private static final long serialVersionUID = 7620582079916035164L;
-	public static final String ADMIN_STR = "admin";
-	private boolean flag_Xmpp;
-	private boolean flag_Sms;
+	private boolean flag_Xmpp = false;
+	private boolean flag_Sms = false;
 
-	private static final Logger LOGGER = LoggerFactory.getLogger(MainMenu.class);
+	private final Logger logger = LoggerFactory.getLogger(MainMenu.class);
 
-	@Override
 	public void loginInserted(AWTEvent e) {
 		if (e.getSource() instanceof User) {
 			myUser = (User) e.getSource();
 			MDC.put("OHUser", myUser.getUserName());
 			MDC.put("OHUserGroup", myUser.getUserGroupName().getCode());
-			LOGGER.info("Logging: \"{}\" user has logged into the system.", myUser.getUserName());
+			logger.info("Logging: \"{}\" user has logged the system.", myUser.getUserName());
```

Comment:
correct form is the one on develop LOGGER and "into"

Reply:
I restored the current informatici version